src/IceNucleation.jl

```diff
@@ -54,4 +55,39 @@ function dust_activated_number_fraction(
     end
 end
 
+"""
+   ABIFM_J(prs, x, T)
+
+ - `prs` - set with model parameters
+ - `x` - wt% sulphuric acid / 100
+ - `T` - air temperature [K]
+
+Returns the immersion freezing nucleation rate coefficient, `J`, in cm^-2 s^-1.
+`m` and `c` constants for illite are taken from Knopf & Alpert 2013 Table 1.
+"""
+function ABIFM_J(
+    prs::APS,
+    x::FT,
+    T::FT
+) where {FT <: Real}
+
+    thermo_params = CMP.thermodynamics_params(prs)
+
+    m = 54.48075    # for illite particles
```

Comment:
We will want to make those parameters part of CLIMAParameters at some point. Maybe for now we can just define them at the top of the file. So that they are easier to move later on